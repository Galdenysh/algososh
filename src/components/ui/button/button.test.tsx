import { Button } from "./button";
import renderer from "react-test-renderer";

test("button with text", () => {
  const tree = renderer.create(<Button text="Button" />).toJSON();
  expect(tree).toMatchSnapshot();
});
