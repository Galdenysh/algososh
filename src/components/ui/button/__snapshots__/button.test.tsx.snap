// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button button disabled 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  disabled={true}
  type="button"
>
  <p
    className="text undefined"
  />
</button>
`;

exports[`button button isLoader 1`] = `
<button
  className="text text_type_button text_color_primary button undefined loader "
  disabled={true}
  type="button"
>
  <img
    alt="Загрузка."
    className="loader_icon"
    src=""
  />
</button>
`;

exports[`button button with text 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  >
    Button
  </p>
</button>
`;

exports[`button button without text 1`] = `
<button
  className="text text_type_button text_color_primary button undefined false "
  type="button"
>
  <p
    className="text undefined"
  />
</button>
`;
